src/Datasource/ModelAwareTrait.php

```diff
@@ -114,4 +129,22 @@ public function modelFactory($type, callable $factory)
     {
         $this->_modelFactories[$type] = $factory;
     }
+
+    /**
+     * Set or get the model type to be used by this class
+     *
+     * @param string|null $modelType The model type or null to retrieve the current
+     *
+     * @return string|$this
+     */
+    public function modelType($modelType = null)
+    {
+        if ($modelType === null) {
+            return $this->_modelType;
+        }
+
+        $this->_modelType = $modelType;
+
+        return $this;
```

Comment:
@Marlinc Most methods don't return `$this`, only the ones which have fluent interface.